<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_rmsd</class>
 <widget class="QDialog" name="dlg_rmsd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compute RMSD </string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="m_pAlign">
        <property name="text">
         <string>Align structures</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel1">
        <property name="text">
         <string>Select:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="m_pSelectedAtoms">
        <item>
         <property name="text">
          <string>All Atoms</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Backbone</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Trace</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Show All</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="m_pCompute">
       <property name="text">
        <string>Compute RMSD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>m_pCompute</tabstop>
  <tabstop>m_pCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_pCompute</sender>
   <signal>clicked()</signal>
   <receiver>dlg_rmsd</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pCancel</sender>
   <signal>clicked()</signal>
   <receiver>dlg_rmsd</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
